c++/src/Compression.cc

```diff
@@ -1204,7 +1245,12 @@ DIAGNOSTIC_PUSH
         (new Lz4CompressionSteam(
           outStream, level, bufferCapacity, compressionBlockSize, pool));
     }
-    case CompressionKind_SNAPPY:
+    case CompressionKind_SNAPPY: {
+      int level = 0;
```

Comment:
Does snappy support different levels? If true, we should assign different levels according to CompressionStrategy

Reply:
There's no mention about it in the header file, and I couldn't find anything elsewhere, that would indicate it does.